docs/versions.json

```diff
@@ -0,0 +1,5 @@
+[
+    "v3.1.0",
```

Comment:
Could you add a reminder to add new releases here to https://github.com/cosmos/interchain-security/blob/main/RELEASE_PROCESS.md?